### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by masesisaac** [o-kamiye/pandabot#1]
https://github.com/Charca/bootbot could save you alot of boilerplate code, its like the bootstrap for messenger bots with nodejs.

**Comment by o-kamiye** [comment]
@maseruka Thanks for the suggestion. Would you help make improve the bot experience if we use `bootbot` framework?

**Comment by masesisaac** [comment]
yeah, am going to first transform the existing codebase to bootbot, submit the code and you test if everything is fine, then we can keep on adding more stuff
